.github/workflows/documentation.yml

```
@@ -0,0 +1,149 @@
+---
+name: Documentation
+
+# Builds Markdown documentation for core and addons.
+
+on:
+  push:
+    branches:
+      - main
+      - releasebranch_*
+  pull_request:
+
```

Would adding a workflow_dispatch trigger be useful for a first iteration?
```suggestion
on:
  push:
    branches:
      - main
      - releasebranch_*
  pull_request:
  workflow_dispatch:

```

---

I can see that this may be useful later due to the grass-addons repo being a dependency of this workflow.